sherpa/models/regrid.py

```diff
@@ -502,12 +409,23 @@ class ModelDomainRegridder1D():
     instead it creates an instance that is used to evaluate
     the model.
 
+    Parameters
+    ----------
+    evaluation_space : object or None, optional
+    name : str, optional
+        The default name is 'regrid1d'.
+    interp : function reference, optional
+        The interpolation function: it should accept three arrays: the
+        output x values and the x, y values to interpolate, and return
+        the interpolated y values. The default is to use
+        sherpa.utils.akima.akima.
+
     Attributes
     ----------
-    method
-        The function that interpolates the data from the internal
-        grid onto the requested grid. The default is akima. This is
-        *only* used for point grids, as integrated grids use a simple
+    method : function reference
```

Comment:
You mean "function", not "function reference" right? A function is an object, so that's the object you would put in. I'm not aware of any "reference" or "pointer" type in python/

Reply:
There should be function references. I thought there was specific terminology for "hey, I'm being all first-order and sending around callable thingies" that was more than just 'function'. Maybe "callable"?

Reply:
"callable" is good. In fact, under the hood a function is just a callable object...